privaterelay/management/commands/sync_phone_related_dates_on_profile.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime, timedelta, timezone
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from privaterelay.fxa_utils import get_phone_subscription_dates
+from privaterelay.management.utils import (
+    get_free_phone_social_accounts,
+    get_phone_subscriber_social_accounts,
+)
+import logging
+
+logger = logging.getLogger("events")
+
+
+def sync_phone_related_dates_on_profile(group: str) -> int:
+    social_accounts = get_phone_subscriber_social_accounts()
+    free_phones_social_accounts = get_free_phone_social_accounts()
+    if group == "free":
+        social_accounts = free_phones_social_accounts
+    if group == "both":
+        social_accounts.update(free_phones_social_accounts)
+
+    if not settings.PHONES_ENABLED or len(social_accounts) == 0:
+        return 0
+
+    num_updated_accounts = 0
+    datetime_now = datetime.now(timezone.utc)
+    for sa in social_accounts:
+        date_subscribed_phone, start_date, end_date = get_phone_subscription_dates(sa)
+        profile = sa.user.profile
+        if (date_subscribed_phone and start_date and end_date) is None:
+            # No subscription info from FxA
+            if group == "subscription":
+                # Unsure if social account user should have phone subscription
+                logger.error(
+                    "no_subscription_data_in_fxa_for_user_with_phone_subscription",
+                    extra={"fxa_uid": sa.uid},
+                )
```

Comment:
The Sentry logger configuration is set so that errors emitted on the loggers are also reported in Sentry (a good example of this already happening is [s3_object_does_not_exist](https://mozilla.sentry.io/issues/3720260066/?environment=prod&project=4503976951152641&query=is%3Aunresolved+s3_object_does_not_exist&referrer=issue-stream&statsPeriod=14d).